src/lib/crypto/kimchi_pasta_snarky_backend/field.ml

```diff
@@ -203,10 +203,16 @@ module Make (F : Input_intf) :
 
       let of_yojson j =
         match j with
-        | `String h ->
-            Ok (of_bigint (Bigint.of_hex_string h))
+        | `String s ->
+            let parsed_bigint =
+              if String.is_prefix ~prefix:"0x" s then Bigint.of_hex_string s
+              else
+                (* NOTE: we're dealing with a older precomputed block *)
```

Comment:
Not sure it is relevant for future devs to mention "with an older precomputed block". Out of the context, it does not mean anything.